### PR TITLE
CIV-6955 Remove PBA data collection

### PR DESCRIPTION
**Pull request by miguelMolina3691**
**Before creating a pull request make sure that:**

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/CIV-6955

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x ] No
```

### CODE REVIEW

**Review by m-meulendijks-v1 — APPROVED** [review]

**Review by hmcts-version1-ignacio — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java [hunk 1]
```diff
@@ -72,7 +86,7 @@ private CallbackResponse makePaymentServiceReq(CallbackParams callbackParams) {
                 caseData = caseData.toBuilder()
                     .claimIssuedPBADetails(SRPbaDetails.builder()
                                                   .applicantsPbaAccounts(caseData.getApplicantSolicitor1PbaAccounts())
-                                                  .fee(caseData.getClaimFee())
+                                                  .fee(fee)
```
Comment (filippo-possenti-version1) [inline src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java:89]: I suspect this bit needs to be behind feature flag.

Can you please check?

**Review by filippo-possenti-version1 — CHANGES_REQUESTED** [review]
I believe there's a bit that needs to be behind feature flag.

File: src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandlerTest.java [hunk 2]
```diff
@@ -65,24 +74,57 @@ public void setup() {
...
+            params = callbackParamsOf(caseData, ABOUT_TO_SUBMIT);
             var response = (AboutToStartOrSubmitCallbackResponse) handler.handle(params);
 
             verify(paymentsService).createServiceRequest(caseData, "BEARER_TOKEN");
             assertThat(extractPaymentDetailsFromResponse(response).getServiceReqReference())
                 .isEqualTo(SUCCESSFUL_PAYMENT_REFERENCE);
         }
 
+        @Test
+        void shouldMakeHearingPaymentServiceRequest_whenInvoked() throws Exception {
```
Comment (filippo-possenti-version1) [inline src/test/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandlerTest.java:104]: Same as above. Given/When/Then comments.

File: src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java (hunk 1 above)
Comment (filippo-possenti-version1) [reply src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/payment/ServiceRequestAPIHandler.java:89]: Nevermind.

**Review by filippo-possenti-version1 — DISMISSED** [review]

**Review by hmcts-version1-ignacio — DISMISSED** [review]

**Comment by filippo-possenti-version1** [comment]
This was green yesterday. Today it's not due to an unrelated issue with dates and bank holidays.
Approvals got removed due to a merge conflict but it's really all good.

I'll be overriding protections and merge this, knowing that the builds will turn red due to the unrelated issue.

**Review by filippo-possenti-version1 — APPROVED** [review]

**Pull request merged**